test/behaviour/mocks/locations.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+	request: {
+		path: '/locations',
+		method: 'GET'
+	},
+	response: {
+		data: [{
+			id: 1,
+			name: 'Sandsäcke füllen',
+			latitude: 49.1232,
+			longitude: 8,
+			location: 'Schlossplatz'
+		}, {
+			id: 2,
+			name: 'Transport',
+			latitude: 48.1232,
+			longitude: 8,
+			location: 'Rheinbad',
+			options: {
+				title: 'Transport im Rheinbad'
+			}
+		}, {
+			id: 2,
```

Comment:
This mock file is not necessary. It was removed in e5b1be6284d40c5426e5b48b3945513aca9da984 .